connector/src/main/java/org/geysermc/connector/network/LoggingPacketHandler.java

```diff
@@ -25,12 +25,17 @@
 
 package org.geysermc.connector.network;
 
+import java.util.ArrayList;
```

Comment:
Non-Java imports should be the first block, followed by Java imports.

Reply:
I'll change it. Editor did it this way, I'll see if I can changed that config.

Reply:
Thanks!